js-excercise/validate/index.js

```diff
@@ -0,0 +1,70 @@
+const form = document.getElementById('form');
+
+form.addEventListener('submit', e => {
+    e.preventDefault();
+
+    const formData = new FormData(form);
+    validateInputs(formData);
+});
+
+const setError = (element, message) => {
+    const inputControl = element.parentElement;
+    const errorDisplay = inputControl.querySelector('.error');
```

Comment:
@quanghuy1799 Please refer this https://developer.mozilla.org/en-US/docs/Web/API/Element/nextElementSibling